contents/uix/design-systems.md

```diff
@@ -0,0 +1,130 @@
+<frontmatter>
+  title: Accessibility
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Design Systems
+
+**Author(s): [Tan Heng Yeow](https://github.com/tanhengyeow)**
+
+## What is a Design System
+
+Design Systems has been a trending topic for the past 2 years. The following definition best encapsulates the essence of a Design System.
```

Comment:
This might become inaccurate with the passage of time. Can be removed without losing any value.